GeoLib/AnalyticalGeometry.h

```diff
@@ -384,6 +384,12 @@ void computeAndInsertAllIntersectionPoints(GeoLib::PointVec &pnt_vec,
 GeoLib::Polygon rotatePolygonToXY(GeoLib::Polygon const& polygon_in,
 	MathLib::Vector3 & plane_normal);
 
+/// Sorts the vector of segments such that the \f$i\f$-th segment is connected
+/// with the \f$i+1\f$st segment, i.e. the end point of the \f$i\f$-th segment
+/// is the start point of the \f$i+1\f$st segment.
+void sortSegments(MathLib::Point3d const& seg_beg_pnt,
```

Comment:
Two questions. maybe you can answer them in the documentation
- why do you need to pass `seg_beg_pnt`? I thought you can automatically determine the beginning point from connectivity of segments.
- do you consider a case when some segments are not connected?